content/docs/deliverability/ip-warm-up-overview.md

```diff
@@ -16,6 +16,17 @@ SparkPost recommends starting off with your best performing messages - those to
 
 During the Warm-up phase the more consistent you are with volume, frequency, complaint and bounce levels, the faster you will establish a positive sending reputation.  If you send infrequently - anything less than weekly - it will take more time to build a positive sender reputation.
 
+### How many IP’s do I need?  
+
+This is a question that we hear a lot and there isn’t a one-size-fits-all answer. Here are a few questions to ask yourself to help you determine just how many IP’s are needed to support your sending program. And if you need more help, please login and create a ticket.
+
+1. How many unique message streams do I have? 
+ * Separating message streams on separate IP’s is a best practice and ensures that the sending activity from your Marketing emails doesn’t impact your ability to get your Transactional emails delivered to the inbox for example. 
+ * Other message stream examples: transactional, marketing, individual brands, regions, etc.
+2. What is the maximum volume that I need to send in a single day for each message stream?
+ * A single IP address can technically send a large volume of mail quickly depending on sending reputation - that’s millions of messages per hour. For most senders the recommended volume after a proper warmup is 2 to 5 million per day from a single IP.
+ * Senders with volume less than 500k messages per month or with inconsistent volume from week to week may not have enough volume to build and maintain a positive sending reputation with mailbox providers. In this case, a shared IP pool may result in better delivery and deliverability. 
```

Comment:
@tcrawford-SP It looks like the formatting is a little off causing it to have two "1."s.  I'll fix that.

<img width="1497" alt="Screen Shot 2021-12-22 at 8 57 39 AM" src="https://user-images.githubusercontent.com/1335605/147104204-4c6d1d28-a37b-4f34-bb73-b79ac5866857.png">

Reply:
I wonder why that happened.